test/jasmine/tests/plots_test.js

```diff
@@ -287,6 +287,17 @@ describe('Test Plots', function() {
                 expect(svgHeight).toBe(400);
             }
         });
+
+        it('should update the axis scales', function() {
+            var fullLayout = gd._fullLayout,
+                plotinfo = fullLayout._plots.xy;
+
+            expect(fullLayout.xaxis._length).toEqual(240);
+            expect(fullLayout.yaxis._length).toEqual(220);
+
+            expect(plotinfo.xaxis._length).toEqual(240);
+            expect(plotinfo.yaxis._length).toEqual(220);
```

Comment:
Got it. Sounds good! I once misunderstood `beforeAll` to mean before each all tests, so thought it was worth checking!